docs/user/syncing-apprepository-webhook.md

```diff
@@ -0,0 +1,159 @@
+> In other platforms the process will be pretty similar. Doubts? Please feel free to [open an issue](https://github.com/kubeapps/kubeapps/issues/new) if you need further guidance!
+
+
+## TL; DR
+
+- Create the ServiceAccount, grant with sufficient permissions and get the token:
+```bash
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: apprepositories-webhook
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: apprepositories-webhook
+rules:
+  - apiGroups:
+      - kubeapps.com
+    resources:
+      - apprepositories
+    verbs:
+      - get
+      - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: apprepositories-webhook
+  namespace: kubeapps
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: apprepositories-webhook
+subjects:
+- kind: ServiceAccount
+  name: apprepositories-webhook
+  namespace: default
+EOF
+```
+
+- Get the `<TOKEN>`:
+```bash
+kubectl get secret $(kubectl get serviceaccount apprepositories-webhook -o jsonpath='{range .secrets[*]}{.name}{"\n"}{end}' | grep apprepositories-webhook) -o jsonpath='{.data.token}' -o go-template='{{.data.token | base64decode}}' && echo
+```
+
+- In your platform:
+	- Webhook URL:    `https://<KUBEAPPS_URL>/api/v1/clusters/<CLUSTER_NAME>/namespaces/<NAMESPACE_NAME>/apprepositories/<APPREPO_NAME>/refresh`
+	- Method: `POST`
+	- Authentication header: `Bearer <TOKEN>`
+
+> For instance: `https://<KUBEAPPS_URL>/api/v1/clusters/default/namespaces/kubeapps/apprepositories/bitnami/refresh` will update the App Repository `bitnami` in the namespace `kubeapps` of the cluster `default`. 
+
+
+
+## Creating and granting a ServiceAccount for authenticating requests
+
+Since the webhook endpoint will be used outside the user browser session (i.e., no OICD login), an ad-hoc ServiceAccount ought to be created; next, it has to be granted with sufficient permissions to `get` and `update` the `AppRepository` objects. Finally, the ServiceAccount token must be retrieved.
+
+### Creating the Kubernetes ServiceAccount, ClusterRole and RoleBinding objects
+
+First, the `ServiceAccount` and the `ClusterRole` objects are created. In this example, the ServiceAccount *apprepositories-webhook* is being created.
+
+```bash
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: apprepositories-webhook
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: apprepositories-webhook
+rules:
+  - apiGroups:
+      - kubeapps.com
+    resources:
+      - apprepositories
+    verbs:
+      - get
+      - update
+EOF
+```
+
+Next, the `ClusterRole` *apprepositories-webhook* (with `get` and `update` permissions over `apprepositories`) has to be associated to a certain (or multiple) namespace(s) by means of a `RoleBinding`:
```

Comment:
Since you give an example of how to bind it in multiple namespaces below, this sentence might be simpler as "...to be bound to the relevant namespace(s) by means of a `RoleBinding`:"